bft/templates/scalar_function.j2

```diff
@@ -72,8 +72,7 @@
             </select>
             {% for dialect in dialects %}
             <dl class="dialect-definition" id="dialect-{{ dialect.name }}">
-                {% for opt, val in dialect.options %}
-                <dt>{{ opt }}</dt>
```

Comment:
Do we know what the output of the new changed code looks like? 
I think in the past this rendered as
```
overflow
    SILENT
rounding
    TIE_TO_EVEN
```

but if we remove the one of the two, won't we end up losing half of the information?

@sanjibansg can you confirm this?

Reply:
Ok I see where was that, thanks for the pointer @amol- . It seems there's something that has to be updated still. If @sanjibansg can't investigate I can't try to understand where was this updated and why:
![image](https://github.com/voltrondata/bft/assets/639755/f36de038-3d84-4b94-82ab-0720fba7e5a1)

Reply:
Ok, I've found this was changed on this PR: https://github.com/voltrondata/bft/pull/8/files#diff-cf88663aeb2bec1d14d86e6e1e443fea4a19e61c72cd146df36ee7fba999c206
Now it renders correctly, see from my local build:
![image](https://github.com/voltrondata/bft/assets/639755/2d1d21e6-b548-487b-912c-a01a1cc8c2b6)